docs/conf/portland/2018/news/volunteers-shirts-tickets.rst

```diff
@@ -0,0 +1,66 @@
+:template: {{year}}/{{templatecode}}/generic.html
+
+.. post:: March 19, 2018
+   :tags: portland-2018, volunteers, shirts, tickets
+
+Call for Volunteers, Shirts on Sale, Tickets Update
+===================================================
+
+Greetings, fellow documentarians! The Portland conference is just short of two months away, and we can't wait to see you all soon! Here are the latest updates from the conference team to help you prepare.
+
+Call for volunteers
+-------------------
+
+There is plenty to do during the conference and the success of the conference relies on our volunteers. The conference organizers are looking for folks who are willing to help out during the conference. If you volunteer for two or more shifts (clocking in at an average of three hours each), you will receive free registration for the conference.
```

Comment:
I'm not mega keen on "success of the conference relies on our volunteers" how about 

volunteers are essential to the sucess of...
we depend on our volunteers to
or even success of the conference depends on our...

And if we're saying our volunteers, we should probably say WE're looking for folks...